include/nbl/config/BuildConfigOptions.h.in

```diff
@@ -19,6 +19,9 @@
 #cmakedefine _NBL_COMPILE_WITH_OPENGL_
 #cmakedefine _NBL_COMPILE_WITH_VULKAN_
 
+// native platform interface
+#cmakedefine _NBL_USE_SYSTEM_EGL
```

Comment:
wrong way to go about it , always compile with EGL... just decide which function pointers to use at runtime